ddl: add admin pause/resume commands (#13294)
pingcap/docs: ddl-introduction.md

    If a completed DDL task is canceled, you can see the `DDL Job:90 not found` error in the `RESULT` column, which means that the task has been removed from the DDL waiting queue.

- `ADMIN PAUSE DDL JOBS job_id [, job_id]`: Used to pause the DDL tasks that are being executed. After the command is executed, the SQL statement that executes the DDL task is displayed as being executed, and the background task is paused.

>     Tasks that are not being executed or waiting cannot be paused. The `Job 3 can't be paused now` error is shown in the `RESULT` column.

Author: ```suggestion
    You can pause only DDL tasks that are in progress or still in queue. Otherwise, the `Job 3 can't be paused now` error is shown in the `RESULT` column.
```

>     You can pause only DDL tasks that are in progress or still in queue. Otherwise, the `Job 3 can't be paused now` error is shown in the `RESULT` column.

Reviewer: ```suggestion
    You can pause only DDL tasks that are in progress or still in the queue. Otherwise, the `Job 3 can't be paused now` error is shown in the `RESULT` column.
```

> - `ADMIN PAUSE DDL JOBS job_id [, job_id]`: Used to pause the DDL tasks that are being executed. After the command is executed, the SQL statement that executes the DDL task is displayed as being executed, and the background task is paused.
> 
>     You can pause only DDL tasks that are in progress or still in the queue. Otherwise, the `Job 3 can't be paused now` error is shown in the `RESULT` column.
> 
> - `ADMIN RESUME DDL JOBS job_id [, job_id]`: Used to resume the DDL tasks that have been paused. After the command is executed, the SQL statement that executes the DDL task is displayed as being executed, and the background task is resumed.

Author: ```suggestion
- `ADMIN PAUSE DDL JOBS job_id [, job_id]`: Used to pause the DDL tasks that are being executed. After the command is executed, the SQL statement that executes the DDL task is displayed as being executed, and the background task is paused. (Experimental feature)

    You can pause only DDL tasks that are in progress or still in the queue. Otherwise, the `Job 3 can't be paused now` error is shown in the `RESULT` column.

- `ADMIN RESUME DDL JOBS job_id [, job_id]`: Used to resume the DDL tasks that have been paused. After the command is executed, the SQL statement that executes the DDL task is displayed as being executed, and the background task is resumed. (Experimental feature)
```

>     Tasks that are not paused cannot be resumed. The `Job 3 can't be resumed` error is shown in the `RESULT` column.

Author: ```suggestion
    You can only resume a paused DDL task. Otherwise, the `Job 3 can't be resumed` error is shown in the `RESULT` column.
```
